content/hi/docs/reference/glossary/pod-lifecycle.md

```diff
@@ -0,0 +1,19 @@
+---
+title: पॉड जीवनचक्र (Pod Lifecycle)
+id: पॉड जीवनचक्र (pod-lifecycle)
+date: 2019-02-17
+full-link: /डॉक्स/कॉन्सेप्ट्स/वर्कलोड्स/पॉड्स/पॉड-लाइफसाइकल/
+related:
+ - पोडो
+ - कंटेनर
+tags:
+ - मौलिक
+short_description: >
+  अवस्थाओं का क्रम जिसके माध्यम से एक पॉड अपने जीवनकाल में गुजरता है।
+ 
+---
+ अवस्थाओं का क्रम जिसके माध्यम से एक पॉड अपने जीवनकाल में गुजरता है।
+
+<!--अधिक-->
+
+[पॉड लाइफ़साइकल](/ डॉक्स/कॉन्सेप्ट्स/वर्कलोड्स/पॉड्स/पॉड-लाइफसाइकल/) को पॉड की अवस्थाओं या चरणों द्वारा परिभाषित किया जाता है। पांच संभावित पॉड चरण हैं: लंबित, दौड़ना, सफल, विफल और अज्ञात। पॉड स्थिति का एक उच्च-स्तरीय विवरण [पॉडस्टैटस] (/ डॉक्स/संदर्भ/जेनरेटेड/कुबेरनेट्स-एपीआई/{{< परम "संस्करण" >}}/# पॉडस्टैटस-वी 1-कोर) `चरण` फ़ील्ड में सारांशित किया गया है। .
```

Comment:
```suggestion
[पॉड जीवनचक्र](/docs/concepts/workloads/pods/pod-lifecycle/) को पॉड की अवस्थाओं या चरणों द्वारा परिभाषित किया जाता है। पांच संभावित पॉड चरण हैं: लंबित, दौड़ना, सफल, विफल और अज्ञात। पॉड स्थिति का एक उच्च-स्तरीय विवरण [पॉडस्टैटस] (/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podstatus-v1-core) `चरण` फ़ील्ड में सारांशित किया गया है। .
```

Reply:
here we are explaining about the stages of the lifecycle of pod. so from opinion we need not to translate these terms also like `pending`,`running`,`Succeeded` and so on. cc @divya-mohan0209, @Garima-Negi

Reply:
as mentioned about this `लंबित, दौड़ना, सफल, विफल और अज्ञात` these are the lifecycle stages of pod so we can do like this `पेंडिंग , रनिंग , सक्सीडेड ` but we will wait for the suggestions from reviewers as well once they will confirm it then you can make changes as i mentioned.